src/features/editor/index.ts

```diff
@@ -0,0 +1,3 @@
+export { default as ContentEditor } from "./ui/ContentEditor";
+export { default as PostTitleInput } from "./ui/PostTitleInput";
```

Comment:
```suggestion
export ContentEditor from "./ui/ContentEditor";
```

이렇게 써도 작동합니다 ㅎㅎ. (default란 변수를 export하는 원리라 그래요.. js가 좀 그렇습니다)

저는 개인적으로는 export default를 사용하기보단 `export const { ... }` 방식을 더 선호하는데요,

1. 오타를 방지할 수 있고
2. 모든 파일이 아니라 개발자가 내보내고 싶은 값들만 선택하여 내보낼 수 있음

위 부분때문에 `export default`를 지양하는 편입니다. (취향이라고 봐주세요 ㅎㅎ)

Reply:
생각하지 못했던 부분인데 짚어 주셔서 감사합니다 !! js가 내부적으로 어떻게 처리하고 있는지도 다시 한 번 봐야겠네요.